hubmap_commons/hm_auth.py

```diff
@@ -465,6 +465,8 @@ def getUserTokenFromRequest(self, httpReq, getGroups):
             return tokenResp        
     
     def getUserInfo(self, token, getGroups = False):
+        if token is None or token.strip() == '':
```

Comment:
In the following use cases when we parse the token by calling `getAuthorizationTokens()`, 
```
token = auth_helper_instance.getAuthorizationTokens(request.headers)
has_write_privs = auth_helper_instance.has_write_privs(token)
has_read_privs = auth_helper_instance.has_read_privs(token)
has_admin_privs = auth_helper_instance.has_data_admin_privs(token)
```
we may get back a `flask.Response` object if [one of the conditions satisfies](https://github.com/hubmapconsortium/commons/blob/1ca9777c70d88c02904b2a5dd185a9ce52f84cfb/hubmap_commons/hm_auth.py#L386-L400), which throws the below error:
```
AttributeError: 'Response' object has no attribute 'strip'
```
@shirey we should also check `isinstance(token, Response)`.